bcda/bcdacli/cli.go

```diff
@@ -362,6 +363,17 @@ func setUpApp() *cli.App {
 			Action: func(c *cli.Context) error {
 				ignoreSignals()
 				success, failure, skipped, err := cclf.ImportCCLFDirectory(filePath)
+				if err != nil {
+					log.API.Error("error returned from ImportCCLFDirectory: ", err)
+					cli.NewExitError("an error occurred during the cclf-import process, see logs for more details", 1)
+					return err
```

Comment:
@laurenkrugen-navapbc where does log.API.Error log to again? (same Q for cli logs)

Reply:
log.API is one of the global loggers that gets setup [here](https://github.com/CMSgov/bcda-app/blob/master/log/logger.go#L34C6-L34C11). The logs get written to a file and ingested into splunk from there!

Reply:
@laurenkrugen-navapbc what about `cli.NewExitError` ? 👀

Reply:
https://pkg.go.dev/gopkg.in/urfave/cli.v1#NewExitError

Since we are using the cli package to have the app intercept cli commands, `cli.NewExitError` is the equivalent of the `bash exit #` command. That actually exits the cli altogether, which is not what we want so I removed it.